### ISSUE TITLE
Dynamic year in footer

### ISSUE THREAD

**Issue by Mehonoshin** [symbiod/symbiod#571]
**Value**
To prevent future issues let's make the year in copyright dynamic.

**Comment by tsrandrei** [comment]
#4a7a81b ?

**Comment by Mehonoshin** [comment]
@tsrandrei I wonder why we have 2018 at the production then https://www.symbiod.org/

**Comment by tsrandrei** [comment]
@Mehonoshin, I have no idea, it's have been merged into master almost 1 month ago e5075ece440d887e03b9e3e5de3f6bd331be0f36

**Comment by Mehonoshin** [comment]
@tsrandrei yeah, sorry, my bad. It looks like our cucumber tests are failing, so it blocks automatic deployments.

**Comment by tsrandrei** [comment]
@Mehonoshin , is because of my last merges? Curious

**Comment by Mehonoshin** [comment]
@tsrandrei I will investigate it, no idea yet
